clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -6410,8 +6377,7 @@ public void testUpdateFeaturesHandleNotControllerException() throws Exception {
                 request -> request instanceof UpdateFeaturesRequest,
                 UpdateFeaturesResponse.createWithErrors(
                     ApiError.NONE,
-                    Utils.mkMap(Utils.mkEntry("test_feature_1", ApiError.NONE),
-                                Utils.mkEntry("test_feature_2", ApiError.NONE)),
+                    Utils.mkSet("test_feature_1", "test_feature_2"),
```

Comment:
Which version of the response are we testing here? If it's the latest version, the set should be empty, right?

Reply:
Ah yeah, I referenced this above. There is no serialization so this is going to respond like a v1 request (we won't ignore the feature fields). 

I can change it to not do this, but I wonder if we have testing for v1 responses now. 🤔

Reply:
Maybe I will add some v1/v2 tests here, and one for the serialization in the UpdateFeaturesResponseTest